styles.css

```diff
@@ -138,32 +134,38 @@
 
   .Auther__BackgroundImg {
      position: relative;
-     display: block;
      margin-left: auto;
      margin-right: auto; 
     
    }
 
    .Auther__BackgroundImg--Sticker1 {
      position: absolute;
-     margin-left: -20px;
-     margin-top: 150px;
+     left: 577px;

```

Comment:
if the component is closer to the right, than the left, it better to use the right instead,